cdap-ui/app/features/datasets/templates/tabs/explore.html

```diff
@@ -60,84 +60,84 @@ <h4 class="panel-title" bs-collapse-toggle>
-              <p>{{ responses.request.statement }}</p>
-            </div>
+            </table>
+          </div>
 
-            <div class="text-right" ng-show="query">
-              <button class="btn btn-primary" disabled>Execute</button>
-              <button class="btn btn-default" disabled>Clone</button>
-            </div>
+          <!-- Right Panel -->
+          <div class="col-xs-12 col-md-6 well">
+            <div ng-show="responses.results.length">
+              <p><strong>{{ responses.request.status }}</strong></p>
+              <div class='sql-statement'>
+                <p>{{ responses.request.statement }}</p>
+              </div>
 
-            <div class="results">
-              <h3>Results</h3>
-              <div class="text-right">
-                <a
-                  class="btn btn-default"
-                  ng-click="download(responses.request)"
-                >
-                  Download All
-                </a>
+              <div class="text-right" ng-show="query">
+                <button class="btn btn-primary" disabled>Execute</button>
+                <button class="btn btn-default" disabled>Clone</button>
               </div>
 
-              <div>
-                <table class="table">
-                  <thead>
-                    <tr>
-                      <th ng-repeat="header in responses.schema">{{ header.name }}</th>
-                    </tr>
-                  </thead>
-                  <tbody>
-                    <tr ng-repeat="results in responses.results">
-                      <td ng-repeat="result in results.columns">{{ result }}</td>
-                    </tr>
-                  </tbody>
-                </table>
+              <div class="results">
+                <h3>Results</h3>
+                <div class="text-right">
+                  <a
+                    class="btn btn-default"
+                    ng-click="download(responses.request)"
+                  >
+                    Download All
+                  </a>
+                </div>
+
+                <div>
+                  <table class="table">
+                    <thead>
+                      <tr>
+                        <th ng-repeat="header in responses.schema">{{ header.name }}</th>
+                      </tr>
+                    </thead>
+                    <tbody>
+                      <tr ng-repeat="results in responses.results">
+                        <td ng-repeat="result in results.columns">{{ result }}</td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </div>
               </div>
             </div>
+            <div ng-hide="responses.results.length" class="text-center">
+              <p>(Select Query)</p>
+            </div>
           </div>
-          <div ng-hide="responses.results.length" class="text-center">
-            <p>(Select Query)</p>
-          </div>
-        </div>
 
 
       </div>
```

Comment:
This explore tab had been refactored into a directive.. you'll have merge conflict with this. The View Queries section especially.